SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -1,3 +1,23 @@
+ServerTreeView {
+	*new { |aServer, bounds=(Rect(128, 64, 400, 400)), interval=0.5|
+		var onClose, window;
+		window = Window.new(name.asString, bounds, scroll:true).front;
+		onClose = aServer.plotTreeView(interval, window.view, { defer {window.close}; });
+		window.onClose = {
+			onClose.value;
+		};
+		^window.view.hasHorizontalScroller_(false).background_(Color.grey(0.9));
+	}
+
+	*closeAll {
```

Comment:
better keep a class variable called `all` that keeps track of the existing views. Otherwise a window accidentally called "ServerTreeView" (e.g. by some existing large system) will be closed, too. And it makes docuemntation complicated (would need to go into `Window` help).

Reply:
Actually, a second thought about this: it is a little bit ambivalent whether "all" is supposed to mean all views that concern a given server or all of them. So maybe just leave out `closeAll`. We don't need it that much and it makes things overly complicated. Other views like `Stethoscope`, also have no closeAll.

Reply:
ok, Done!
https://github.com/supercollider/supercollider/pull/6294/commits/1bc7706c859daafa68f26caf704e501f8ad8c5d4